src/core/namespace.c

```diff
@@ -599,11 +599,26 @@ static int clone_device_node(const char *d, const char *temporary_mount) {
         dn = strjoina(temporary_mount, d);
 
         mac_selinux_create_file_prepare(d, st.st_mode);
-        r = mknod(dn, st.st_mode, st.st_rdev);
+        if (do_mknod)
```

Comment:
a bit of a misnomer now, no? I mean, in both cases we invoke mknod() after all. Maybe call this "make_devnode" or so

Reply:
addressed